stl/inc/algorithm

```diff
@@ -1261,6 +1261,43 @@ namespace ranges {
 
     inline constexpr _Copy_if_fn copy_if{_Not_quite_object::_Construct_tag{}};
 } // namespace ranges
+
+namespace ranges {

```

Comment:
We can fuse a little harder here since this is closing and then immediately reopening the same namespace:
```suggestion
```